gnarly.rb

```diff
@@ -314,7 +304,24 @@ def react?
 end
 
 def post_bundle
+  # This is a really unfortunate, but necessary line of code that resets the
+  # cached Gemfile location so the generated application's Gemfile is used
+  # instead of the generators Gemfile.
+  ENV["BUNDLE_GEMFILE"] = nil
+
+  add_gems
```

Comment:
Adding gems *after* bundling, and then bundle again, in particular, seems....weird. What's the story here?

Reply:
Oh, sorry, this isn't after bundle (`after_bundle`), but is in the `post_bundle` method. Perhaps we should rename `post_bundle`?

Reply:
Yeah, sorry. This worked locally but is having trouble on CI so I'm doing more investigating on these issues.

Reply:
So this will almost certainly change before it's good to go again

Reply:
Perhaps `post_bundle` should just be named `create_gnarly_rails_app`, removing the other?